core/platform/models.py

```diff
@@ -33,9 +33,21 @@ def import_models(cls):
 
 
 class _Gae(_Platform):
+    """Provides platform-specific imports related to GAE (Google App Engine).
+    """
     @classmethod
     def import_models(cls, model_names):
+        """Imports and returns the storage models listed in model_names.
 
+        Args:
+            model_names: list(str). List of storage model names.
+
+        Returns:
+            tuple(ndb.Model): Tuple of Storage models.
```

Comment:
Actually, sorry -- on reading this method more carefully, it looks like what's being returned are modules, not models. So this docstring should be similar to the ones below.